### PR TITLE
Metagoofil improvement

### PR DESCRIPTION
**Pull request by zkkok**
# Changes

- **File Download:** Added the functionality to download files that are found
- **Removed Unnecessary Requirements:** Removed required fields for search and file limits
- **Minor Spelling Errors:** Fixed some typos in the user guides
- **Fixed Syntax:** The code was previously running on "cewl", changed to "metagoofil" accordingly

Link to issue #414

### CODE REVIEW

File: src/components/metagoofil/metagoofil.tsx [hunk 1]
```diff
@@ -85,15 +88,21 @@ export function Metagoofil() {
         setLoading(true);
 
         const args = [`-d`, `${values.webname}`, `-t`, `${values.filetype}`];
-
+        //number of searches made
```
Comment (Sakada-lim) [inline src/components/metagoofil/metagoofil.tsx:91]: if statement is fine but maybe hard to scale later down the line. Maybe consider a new logic that can scale for the future.

**Review by Sakada-lim — APPROVED** [review]
code is good. maybe consider scalability for future update

**Comment by zkkok** [comment]
@gangamarx I think you are performing the test on the old code 
image
I can see "cewl" on the output but the new code has switched to "metagoofil"

**Comment by gangamarx** [comment]
Thank you zkkok, yes you are correct, and I was using the old code. 
Now I have updated the code and checked with all parameters as stated and it works well without errors both in CLI and GUI mode (both standard and manual) as per below images. 

GUI Works -std
GUI Works-manual

**Review by Phaellynn — APPROVED** [review]
Metagoofil has passed all functionality tests as tested by Ganga Marx
- Grammatical Errors in ToolTip are fixed
- Document Download Error is now fixed
Screenshot 2023-12-13 133419

**Review by gangamarx — APPROVED** [review]
The tool now works as expected without any errors.

**Pull request merged**